sql/pr/avg-time-to-first-review.sql

```diff
@@ -0,0 +1,21 @@
+-- Average time to first review
+WITH first_reviews AS (
+    SELECT
+        pulls.number as pull_number,
+        pulls.created_at as created_at,
+        min(reviews.submitted_at) as first_review
+    FROM pulls
+    LEFT JOIN reviews
+    ON pulls.number = reviews.pull_number
+    WHERE pulls.owner = 'trinodb' AND pulls.repo = 'trino'
+    GROUP BY 1, 2)
+SELECT
+    date_trunc('month', date(created_at)) as month_created,
```

Comment:
This is the same problem we have with `mean time to merge` - should we group by PR create or merge date. 
We group PRs at all to capture the change in this metric over time, otherwise, we should just grab the mean from the last 3 months. But the same question remains - last 3 months of what? Definitely not creation time, since there are _a lot_ of PRs that live longer than that. I'd pick PRs that had any activity in the last 3 months, that's why I suggested `updated_at`. It doesn't matter where it falls relative to the first (or last) review or creation time.

You don't have to worry about multiple updates on the PR if you use the `unique_pulls` view.